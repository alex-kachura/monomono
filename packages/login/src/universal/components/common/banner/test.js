import React from 'react';
import Banner from './';

describe('[Component: Banner]', () => {
  let component;

  const mockProps = {
    getLocalePhrase: (key) => key,
  };

  beforeEach(() => {
    component = global.contextualShallow(<Banner {...mockProps} />);
  });

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
