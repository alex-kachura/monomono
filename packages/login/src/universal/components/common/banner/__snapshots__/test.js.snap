// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component: Banner] should render correctly 1`] = `""`;
